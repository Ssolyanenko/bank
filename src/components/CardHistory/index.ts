export * from './History';
export * from './Pagination';
export * from './CardHistory';
