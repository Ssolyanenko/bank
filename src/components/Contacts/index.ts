export * from './Contacts';
export * from './ContactsFooter';
export * from './ContactsInformation';
