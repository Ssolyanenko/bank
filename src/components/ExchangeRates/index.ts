export * from './ExchangeRates';
export * from './Converter';
export * from './Rates';
