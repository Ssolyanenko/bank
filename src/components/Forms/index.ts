export * from './Login';
export * from './PasswordRecoveryForm';
export * from './GeneralInfo';
export * from './Security';
export * from './OrderDebitTravelCard';
export * from './SetPinCodeForm';
export * from './OrderCreditCardPremium';
export * from './FormNotifications';
export * from './FilterHistoryForm';
export * from './CardLimitForm';
export * from './OrderCreditCardBilliable';
export * from './OrderDebitSmartCard';
export * from './TransferByPhoneNumberForm';
export * from './AcceptTerms';
