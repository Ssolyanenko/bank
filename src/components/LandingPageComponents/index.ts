export * from './LandingPageHeader';
export * from './LandingPageMain';
export * from './LandingPageFooter';
