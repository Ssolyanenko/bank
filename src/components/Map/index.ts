export * from './MapButton';
export * from './CityModal';
export * from './ConfirmCityModal';
export * from './Sidebar';
export * from './MapPage';
