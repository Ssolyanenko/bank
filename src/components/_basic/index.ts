export * from './PrimaryButton';
export * from './Button';
export * from './Input';
export * from './InputPassword';
export * from './ModalWrapper';
export * from './ModalConfirmation';
export * from './PhoneCodeTimer';
export * from './ModalBox';
export * from './CloseButton';
export * from './BackButton';
export * from './StepperForm';
export * from './StepperLabel';
export * from './ChatButton';
export * from './BanTimer';
export * from './CardTemplate';
export * from './SecondaryButton';
export * from './generic';
export * from './SelectStyled';
export * from './TextFieldStyled';
export * from './InputStyled';
export * from './CardTemplate';
export * from './Modal';
export * from './SecureWrapper';
export * from './InputRange';
export * from './DropDown';
export * from './RadioButtonsGroup';
export * from './FormError';
export * from './Checkbox';
export * from './Switcher';
export * from './EditButton';
export * from './CardBlock';
export * from './Tabs';
export * from './ModalSuccess';
export * from './DropDownWithCheckbox';
export * from './SelectFormikTemplate';
export * from './Loader';
export * from './DatePicker';
export * from './CustomRoutes';
