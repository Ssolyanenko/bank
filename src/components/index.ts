export * from './LandingPageComponents';
export * from './Forms';
export * from './ArrowsBlock';
export * from './RootModal';
export * from './Slider';
export * from './Map';
export * from './Contacts';
export * from './MainPageComponents';
export * from './ExchangeRates';
export * from './CardProducts';
export * from './CardHistory';
export * from './SetPinSuccessModal';
export * from './MyCardsApplications';
export * from './MyLoans';
export * from './MyCardsApplications';
export * from './InformationBlock';
export * from './StatusTag';
export * from './DropDownWithSearchBranches';
export * from './FormInputBox';
export * from './MyCards';
export * from './ActivateBlockCardModal';
export * from './UserAccount';
export * from './SetPinCodeFormModal';
export * from './CardDetails';
export * from './CardStatement';
export * from './CardStatementButtons';
export * from './Accordion';
export * from './ReceiptButtons';
export * from './TemporaryComponent/TemporaryComponent';
export * from './TransferModal';
export * from './ModalChoosePaymentCard';
export * from './CardInfo';
export * from './ShareReceiptModal/ShareReceiptModal';
export * from './SocialMedia';
export * from './ResetFormsConfirmModal';
export * from './TransactionLimit';
export * from './ChangeTransactionLimit';
export * from './ActivateBlockCard';
export * from './CardManage';
export * from './ChangePINCode';
export * from './ResponseModal';
export * from './TransactionLimit';
