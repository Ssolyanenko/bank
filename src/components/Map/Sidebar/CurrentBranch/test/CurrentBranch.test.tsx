import React from 'react';
import { shallow } from 'enzyme';

import { CurrentBranch } from 'components';
import { mockBranch } from 'constants/branch';

describe('<CurrentBranch />', (): void => {
  test('should make a snapshot', (): void => {
    const props = { branch: mockBranch, calculateRoute: jest.fn(), distance: '100m', handleActiveMarker: jest.fn() };
    const wrapper = shallow(<CurrentBranch {...props} />);
    const component = wrapper.getElement();

    expect(component).toMatchSnapshot();
  });
});
