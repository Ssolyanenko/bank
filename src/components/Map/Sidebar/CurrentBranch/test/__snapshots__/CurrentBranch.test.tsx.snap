// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CurrentBranch /> should make a snapshot 1`] = `
<div
  className="branch"
>
  <div
    className="branchInform"
  >
    <BranchContainer
      branch={
        {
          "address": "2 Warple Way",
          "cashWithDrawal": true,
          "city": "London",
          "consultation": false,
          "currencyExchange": false,
          "exoticCurrencyExchange": false,
          "id": 1,
          "insurance": false,
          "latitude": 51.5498969,
          "longitude": -0.1535568,
          "moneyTransfer": true,
          "number": 11,
          "operationModes": [
            {
              "closingTime": "00:00Z",
              "dayOfWeek": "MONDAY",
              "openingTime": "00:00Z",
            },
          ],
          "pandus": false,
          "payment": true,
          "topUp": true,
          "topUpWithoutCard": true,
          "type": "ATM",
        }
      }
      distance="100m"
      selectBranch={[MockFunction]}
    />
    <div
      className="work"
    >
      <span
        className="text"
      >
        Working hours
      </span>
      <div
        className="weekDay"
      >
        <button
          className="buttonDay"
          id="Monday"
          onClick={[Function]}
        >
          Mon
        </button>
        <button
          className="buttonDay"
          id="Tuesday"
          onClick={[Function]}
        >
          Tue
        </button>
        <button
          className="buttonDay"
          id="Wednesday"
          onClick={[Function]}
        >
          Wed
        </button>
        <button
          className="buttonDay"
          id="Thursday"
          onClick={[Function]}
        >
          Thu
        </button>
        <button
          className="buttonDay"
          id="Friday"
          onClick={[Function]}
        >
          Fri
        </button>
        <button
          className="buttonDay"
          id="Saturday"
          onClick={[Function]}
        >
          Sat
        </button>
        <button
          className="buttonDay"
          id="Sunday"
          onClick={[Function]}
        >
          Sun
        </button>
      </div>
      <div
        className="workingHour"
      >
        <OpenNowIcon
          className="icon"
        />
        24/7
      </div>
    </div>
    <div
      className="services"
    >
      <span
        className="text"
      >
        Main services
      </span>
      <div
        className="service"
      >
        <span
          className="operation"
        >
          <GetIcon />
        </span>
        <span>
          CashWithDrawal
        </span>
      </div>
      <div
        className="service"
      >
        <span
          className="operation"
        >
          <TransferIcon />
        </span>
        <span>
          MoneyTransfer
        </span>
      </div>
      <div
        className="service"
      >
        <span
          className="operation"
        >
          <TopOnIcon />
        </span>
        <span>
          TopUp
        </span>
      </div>
      <div
        className="service"
      >
        <span
          className="operation"
        >
          <TopOnWithoutCardIcon />
        </span>
        <span>
          TopUpWithoutCard
        </span>
      </div>
      <div
        className="service"
      >
        <span
          className="operation"
        >
          <PayIcon />
        </span>
        <span>
          Payment
        </span>
      </div>
    </div>
  </div>
  <div
    className="makeRoute"
  >
    <button
      className="build"
      onClick={[MockFunction]}
    >
      Build a route
    </button>
  </div>
</div>
`;
